index.html

```diff
@@ -15,7 +15,7 @@
                     <div class="nav__hamburger-icon-second-line"></div>
                     <div class="nav__hamburger-icon-third-line"></div>
                 </div>
-                <div id="nav__hamburger-menu" class="nav__hamburger-menu --off">
+                <div id="nav__hamburger-menu" class="nav__hamburger-menu --hidden">
```

Comment:
@YuraLeo94 By BEM if class name used for a state, does it needs to have -- in front?

Reply:
It has a very interesting naming for the state, which has the next syntax: `block__element--modifications` or `element--modifications`.
![image](https://user-images.githubusercontent.com/71021726/158238210-c9c7df97-1c62-42c7-9914-9e348077042c.png)
links - [getbem.com/naming/](getbem.com/naming/)

But we can come to a different agreement if you'd like.